.vscode/settings.json

```diff
@@ -7,5 +7,8 @@
   "files.insertFinalNewline": true,
   "files.trimFinalNewlines": true,
   "files.trimTrailingWhitespace": true,
-  "githubPullRequests.ignoredPullRequestBranches": ["master"]
+  "githubPullRequests.ignoredPullRequestBranches": [
+    "master"
+  ],
+  "git.ignoreLimitWarning": true
```

Comment:
@semorrison Is this intentional?

Reply:
No, it wasn't, I didn't realise I did that (nor did I make the edit manually...)